src/app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\PostRequest;
+use App\Models\Post;
+
+class PostController extends Controller
+{
+    const DESTINATION_PATH = 'public/images';
+
+    public function create(PostRequest $request)
```

Comment:
shouldn't this method named store?

Reply:
I have resolved this comment, sir. Thank you! 🙇